ReactSkia/components/RSkComponentImage.cpp

```diff
@@ -155,8 +159,15 @@ sk_sp<SkImage> RSkComponentImage::getLocalImageData(string sourceUri) {
 }
 
 inline string RSkComponentImage::generateUriPath(string path) {
-  if(path.substr(0, 14) == "file://assets/")
+  if(path.substr(0, 14) == "file://assets/"){
     path = "./" + path.substr(7);
+  } else if(path.substr(0,4) == "http"){
+
+  } else {
+
+    path =  GENERATEPATH(path); 
```

Comment:
This else condition may not be right if the path is something like "file://xxxxx"